### PR TITLE
Docker on windows

### PR DESCRIPTION
**Pull request by achamayou**
Fix for #351, as kindly suggested by @letmaik

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or the definition README at
+// https://github.com/microsoft/vscode-dev-containers/tree/master/containers/docker-existing-dockerfile
+{
+	// See https://aka.ms/vscode-remote/devcontainer.json for format details.
+	"name": "Existing Dockerfile",
```
Comment (eddyashton) [inline .devcontainer/devcontainer.json]: We should give this a better name

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or the definition README at
+// https://github.com/microsoft/vscode-dev-containers/tree/master/containers/docker-existing-dockerfile
+{
+	// See https://aka.ms/vscode-remote/devcontainer.json for format details.
```
Comment (eddyashton) [inline .devcontainer/devcontainer.json]: Minor: Comments in json are non-standard. I think they'll cause issues in any editor other than VSCode (the syntax highlighting on GitHub doesn't like them too). Can we reduce this to just the JSON, and add an adjacent README pointing at the docs if necessary?

**Comment by cimetrics** [comment]
images

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (achamayou) [reply .devcontainer/devcontainer.json]: Sure, this was generated by VS Code, I'll scrub it.

Comment (achamayou) [reply .devcontainer/devcontainer.json]: Done

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (achamayou) [reply .devcontainer/devcontainer.json]: Done

**Comment by cimetrics** [comment]
images

**Pull request merged**